src/components/tutorial-meta/components/variant-list/variant-list.module.css

```diff
@@ -0,0 +1,67 @@
+.root {
+	margin-top: 24px;
+	margin-bottom: 24px;
+	display: none;
+
+	@media (--dev-dot-tablet-up) {
+		display: block;
+	}
+}
+
+.label {
+	composes: hds-typography-body-200 from global;
+	composes: hds-font-weight-semibold from global;
+	color: var(--token-color-foreground-strong);
+	margin-bottom: 8px;
+	display: block;
+}
+
+.list {
+	list-style: none;
+	margin: 0;
+	background: var(--token-color-surface-faint);
+	border: 1px solid var(--token-color-border-faint);
+	border-radius: 6px;
+	padding: 8px;
+	display: flex;
+	gap: 8px;
+	flex-wrap: wrap;
+}
+
+/**
+* Shares some similar styles with `button-link` but is unique enough
+* to warrant its own styles to avoid overrides
+*/
+
+.link {
+	composes: hds-typography-body-100 from global;
+	composes: hds-font-weight-medium from global;
+	composes: g-focus-ring-from-box-shadow from global;
+	display: flex;
+	border-radius: 5px;
+	border: 1px solid;
+	padding: 6px 12px;
+	box-shadow: var(--token-elevation-low-box-shadow);
+}
+
+.activeLink {
+	background-color: var(--token-color-foreground-action);
+	border-color: var(--token-color-foreground-action-hover);
+	color: var(--token-color-foreground-high-contrast);
+	cursor: not-allowed;
+	box-shadow: none;
```

Comment:
We need a lighter color for the box-shadow here, working to get that info from @braydenlove now.